bc_obps/compliance/tests/service/test_supplementary_version_service.py

```diff
@@ -342,63 +342,6 @@ def test_handle_supplementary_version_no_previous_version(
         mock_decreased_credit_handler.assert_not_called()
 
     # THE FOLLOWING TWO TESTS WILL NEED REWRITING AFTER HANDLING SCENARIOS WHERE CREDITS HAVE BEEN ISSUED/REQUESTED
-    # def test_handle_increased_credit_success(
```

Comment:
Moved this into the increased handler class and renamed to `test_calls_correct_handler`